### PR TITLE
Add new generated protobuf version pb.h/cc

### PR DESCRIPTION
**Pull request by hewigovens**

### CODE REVIEW

File: TrustWalletCore.podspec [hunk 1]
```diff
@@ -18,129 +18,103 @@ Pod::Spec.new do |s|
...
+    "#{protobuf_source_dir}/src/google/protobuf/util/json_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/message_differencer.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/time_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/util/type_resolver_util.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wire_format.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wire_format_lite.cc",
+    "#{protobuf_source_dir}/src/google/protobuf/wrappers.pb.cc",
   s.exclude_files =
     'trezor-crypto/src/rand.c',
-    'trezor-crypto/include/TrezorCrypto/**/base58.h'
```
Comment (vikmeup) [inline TrustWalletCore.podspec]: exclude   'trezor-crypto/include/TrezorCrypto/**/base58.h' ?

Comment (hewigovens) [reply TrustWalletCore.podspec]: preserve_paths and exclude_files can't work at the same time, this is reverted old settings, I found another workaround to fix this

Comment (hewigovens) [reply TrustWalletCore.podspec]: see https://github.com/trustwallet/ios/commit/1f1e14060dbee7f8108ab972b41317993ac4f1ca

**Comment by catenocrypt** [comment]
The Tron and Ziliqa generated source files should excluded from  the git repo.

**Pull request merged**